src/components/ui/Icon/README.md

```diff
@@ -0,0 +1,188 @@
+An `<Icon>` is a graphical representation of a concept, used to illustrate or clarify text, improve wayfinding, and communicate visually.
+
+```jsx
+<Icon name="airplay" />
+<Icon name="aperture" title="Camera Aperture" />
+<Icon name="anchor" title="Boat anchor" />
+```
+
+## Usage
+
+### Use an `Icon` for...
+
+Use an `Icon` when you want to emphasise or clarify an important piece of text. Generally an `Icon` is used with a `Button` or `Tab` to communicate an action or navigation link, but can be used as a standalone element as well. When used as standalone elements, they are often seen in page headers, section titles, in notifications, and inline with text to add clarity.
+
+### Don't use an `Icon` for...
+
+- Displaying a complex image or photo. Use an `Image` instead.
+- Communicating a link, button, or action. Use a `Button` or `Tab` (combined with an `Icon`) instead.
+
+## Appearance
+
+### Colour
+
+An icon can be passed either a stroke or a fill colour.
+
+```jsx
+<Icon name="pen-tool" strokeColor="#053e6c" large/>
+<Icon name="target" strokeColor="#8a0000" large/>
+<Icon name="star" strokeColor="#eeb600" large/>
+<br />
```

Comment:
Is the `<br />` in here really neccessary? It's so ugly! 😢

Reply:
It makes the frontend look better! Is there a better way to split into two lines? I could use `<div>`s, but that seems like a lot of work.

Reply:
Sorry, the reply below (https://github.com/octopusthink/nautilus/pull/88#issuecomment-506910158) resolved this, but emails to GitHub notifications don't get put in the right spot.